### PR TITLE
Update issue template using  Github form schema

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Description

Github expose a beta feature to allow issue template using Github form schema. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.

Besides template update, the change includes
* Add template for question
* Add template for doc issue
* Add survey link to template of issue, question, and feature request
* disable option which allows user to open a blank issue
* Add links of PowerShell repo and MSGraph repo

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3-AZ-QUESTION.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: 🤔 Az module question or discussion
```
Comment (isra-fel) [inline .github/ISSUE_TEMPLATE/3-AZ-QUESTION.yml:1]: If we want to promote https://github.com/Azure/azure-powershell/discussions , we can change this "az module question or discussion" template to a "contact_links"

Comment (dingmeng-xue) [reply .github/ISSUE_TEMPLATE/3-AZ-QUESTION.yml:1]: Bot doesn't support notify service team when labeling service attention on discussion item. Another limitation is author(reporter) will be changed when I transfer discussion to issue. So, for the time being, I'm hesitating to promote discussion.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: General PowerShell Question
+    url: https://github.com/PowerShell/PowerShell/issues/
+    about: PowerShell module or language related question
+  - name: Microsoft Graph Module Question
+    url: https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/
+    about: Question about Microsoft Graph PowerShell Module
```
Comment (isra-fel) [inline .github/ISSUE_TEMPLATE/config.yml:8]: How about linking to autorest.powershell repo?
```suggestion
    about: Question about Microsoft Graph PowerShell Module
  - name: AutoRest PowerShell Generator Question
    url: https://github.com/Azure/autorest.powershell/issues
    about: Questions about the PowerShell plugin of AutoRest
```

Comment (dingmeng-xue) [reply .github/ISSUE_TEMPLATE/config.yml:8]: Added

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**